playground/src/kata.tsx

```diff
@@ -42,15 +84,34 @@ export function Kata(props: {
 
     props.kata.sections.forEach((section, idx) => {
       const parentDiv = itemContent.current[idx];
-      const div = parentDiv?.querySelector(".kata-item-content");
-      if (!div) return;
-      if (section.type === "text") {
-        div.innerHTML = section.contentAsHtml;
-      } else if (section.type === "exercise") {
-        div.innerHTML = section.solutionAsHtml;
-      } else {
-        div.innerHTML = "";
+      let titlePrefix = "\u{1F41B}";
```

Comment:
What are these unicode characters? Can you add a comment so folks don't need to go look them up? (Or better yet, just use the actual character. This is utf-8 after all, no need to restrict to ASCII).

Reply:
Using the characters directly now.